nuxt.config.js

```diff
@@ -150,7 +150,9 @@ export default defineNuxtConfig({
             // eslint-disable-next-line
```

Comment:
The problem I have now is it doesn't set the `process.env.ENABLE_MODERATION_PANEL` I have it set to true but returns undefined so it doesn't set the env variable. My guess is that the nuxt config file just ignores our .env file

Reply:
Solved!